src/tiledb/cloud/bioimg/__init__.py

```diff
@@ -1,4 +1,6 @@
 from tiledb.cloud.bioimg.exportation import export
 from tiledb.cloud.bioimg.ingestion import ingest
 
-__all__ = ("ingest", "export")
+from .types import SupportedExtensions
```

Comment:
do we ever expect users to use this? it seems like this doesn't really need to be exported.

Reply:
Not really nice catch.